### PR TITLE
Changes to align with our GitHub workflow

### PR DESCRIPTION
**Pull request by MMulthaupt**

### CODE REVIEW

**Comment by dragetd** [comment]
Could you take a look at #54 ?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,15 +43,26 @@ To create a release for the latest version:
 ```
 git checkout develop
 git pull
+# check with git log that the branch is in the state which you want to release
 git checkout master
 git pull
-git merge develop --ff-only
```
Comment (dragetd) [inline CONTRIBUTING.md]: I am not sure myself if we really want to force it. Leaving it out creates a fast-forward merge anyways if we worked cleanly. Adding this option makes the merge fail if there is no fast-forward possible. If we want FFs, I'd prefer leaving it in the contributing file.

Comment (MMulthaupt) [reply CONTRIBUTING.md]: My understanding was `--ff-only` is no longer possible in our scenario?

**Comment by MMulthaupt** [comment]
I just noticed, we don't currently explain what our maintenance branches are called. Currently, we have branch protection rules for these branches: 
branches
We have not written down what exactly to call these maintenance branches, e.g. `maint-v1.3.x` or `maint-v1.3` etc.

File: CONTRIBUTING.md (hunk 1 above)
Comment (dragetd) [reply CONTRIBUTING.md]: Why would that be? We merge everything into master, which would be only FFs.

**Comment by dragetd** [comment]
Yep, resolving this is part of #8 :-)

I personally usually use `maint-v1.3.x`, but would be equally okay with `maint-v1.3`. Please choose or roll a dice. :)

**Pull request merged**